python/ray/serve/schema.py

```diff
@@ -1119,3 +1119,19 @@ def _get_user_facing_json_serializable_dict(
                     )
 
         return values
+
+    def _to_deploy_schema(self) -> Optional[ServeDeploySchema]:
+        if self.controller_info is None:
+            return None
```

Comment:
Why do we return `None` if `self.controller_info is None`?

Reply:
Because if controller_info is none, Serve isn't started on this cluster, so there is no config deployed.

Reply:
Could you add that as a comment?

Reply:
Added!